src/test/java/com/thomashan/coup/action/PutBackCardActionTypeTest.java

```diff
@@ -0,0 +1,21 @@
+package com.thomashan.coup.action;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+public class PutBackCardActionTypeTest {
+    @Test
+    public void testIsChallengeable() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Found 'UR'-anomaly for variable 'putBackCardActionType' (lines '9'-'11').](https://app.codacy.com/app/thomashan/coup-java/pullRequest?prid=4561385)